advanced_functionality/xgboost_bring_your_own_model/xgboost_bring_your_own_model.ipynb

```diff
@@ -1,19 +1,18 @@
 {
```

Comment:
<p>Looks like it failed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2791/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
I should have closed this PR. The notebook is fixed in 
https://github.com/aws/amazon-sagemaker-examples/pull/2769